var/spack/repos/builtin/packages/openmc/package.py

```diff
@@ -0,0 +1,88 @@
+##############################################################################
+# Copyright (c) 2013-2016, Lawrence Livermore National Security, LLC.
+# Produced at the Lawrence Livermore National Laboratory.
+#
+# This file is part of Spack.
+# Created by Todd Gamblin, tgamblin@llnl.gov, All rights reserved.
+# LLNL-CODE-647188
+#
+# For details, see https://github.com/llnl/spack
+# Please also see the LICENSE file for our notice and the LGPL.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License (as
+# published by the Free Software Foundation) version 2.1, February 1999.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+
+from spack import *
+import os
+
+
+class Openmc(MakefilePackage):
+    """The OpenMC project aims to provide a fully-featured Monte Carlo particle
+       transport code based on modern methods. It is a constructive solid
+       geometry, continuous-energy transport code that uses ACE format cross
+       sections. The project started under the Computational Reactor Physics
+       Group at MIT."""
+
+    homepage = "https://github.com/ANL-CESAR/"
+    url = "https://github.com/ANL-CESAR/openmc.git"
+
+    tags = ['proxy-app']
+
+    version('develop', git='https://github.com/ANL-CESAR/openmc.git')
+
+    @property
+    def build_targets(self):
+
+        targets = ['--directory=src']
+
+        if self.compiler.name == 'gcc':
+            targets.append('COMPILER=gnu')
+            targets.append('MACHINE=UNKNOWN')
+        if self.compiler.name == 'intel':
+            targets.append('COMPILER=intel')
+        if self.compiler.name == 'pgi':
+            targets.append('COMPILER=pgi')
+        if self.compiler.name == 'xl':
+            targets.append('COMPILER=ibm')
+        if self.compiler.name == 'ftn':
+            targets.append('COMPILER=cray')
+
+        return targets
+
+    def install(self, spec, prefix):
+        with working_dir('src'):
```

Comment:
So the Makefile lives in `src`, but you don't seem to be using this during `build()`. Or is that what `--directory=src` is for? The better way to handle this is with:
```python
build_directory = 'src'
```